synapse/lib/webapp.py

```diff
@@ -87,6 +87,118 @@ def _sendHttpResp(self, code, headers, retinfo):
         self.write(retinfo)
         self.finish()
 
+
+class CrudHand(BaseHand):
+    """Handle create, read, update and delete operations for a given core.
```

Comment:
For consistency with the rest of the code base, plz use single rather than double quotes and always use a newline for doc strings ( which gets stripped off for pydoc generation anyway ).  :)